src/models/MeasureSchemaValidator.ts

```diff
@@ -0,0 +1,27 @@
+import * as Yup from "yup";
+import { Model } from "@madie/madie-models/dist/Model";
+import { MeasureScoring } from "@madie/madie-models/dist/MeasureScoring";
+
+export const MeasureSchemaValidator = Yup.object().shape({
+  measureName: Yup.string()
+    .max(500, "A measure name cannot be more than 500 characters.")
+    .required("A measure name is required.")
+    .matches(/[a-zA-Z]/, "A measure name must contain at least one letter.")
+    .matches(/^((?!_).)*$/, "Measure Name must not contain '_' (underscores)."),
```

Comment:
Minor editorial request: Do you mind making "Measure Name" have consistent captialization in these validation messages, please?